### PR TITLE
Fixed todo's and cleaned up code

### PR DESCRIPTION
**Pull request by robinpiets**
Removed old TODO's, fixed some existing ones and did some regular cleanups

### CODE REVIEW

File: packages/client/src/components/Location/LocationInput.tsx [hunk 1]
```diff
@@ -53,7 +53,6 @@ const LocationInput = ({
       matomoTrackEvent({
         action: actions.CLICK_INTERNAL_NAVIGATION,
         name: `${eventNames.FORWARD} ${
-          // @TODO: there's a small bug here with directly going to OUTCOME when there's no questions to render
```
Comment (robinpiets) [inline packages/client/src/components/Location/LocationInput.tsx:56]: This trackevent is not relevant anymore to fix.

File: packages/client/src/components/Location/LocationSummary.test.tsx [hunk 2]
```diff
@@ -171,90 +169,4 @@ describe("LocationSummary", () => {
       ).not.toBeInTheDocument();
     });
   });
-
-  /**
-   *
-   * OLO FLOW START HERE:
-   *
-   */
-
```
Comment (robinpiets) [inline packages/client/src/components/Location/LocationSummary.test.tsx:180]: All these tests were already covered by my OloLocationInput and OloLocationResult tests

File: packages/client/src/components/Outcome/NewCheckerModal.test.tsx [hunk 3]
```diff
@@ -51,7 +51,6 @@ describe("NewCheckerModal", () => {
     expect(queryByText("Annuleer")).toBeInTheDocument();
 
     // Make sure Matomo Analytics work
-    // TODO: temp disabled this matamo tracking test
```
Comment (robinpiets) [inline packages/client/src/components/Outcome/NewCheckerModal.test.tsx:54]: irrelevant

File: packages/client/src/components/Outcome/NewCheckerModal.test.tsx [hunk 4]
```diff
@@ -98,7 +97,5 @@ describe("NewCheckerModal", () => {
         name: eventNames.OPEN_MODAL_DO_ANOTHER_CHECK,
       });
     });
-
-    // @TODO: This test could be extended to verify that the SessionData has been updated
```
Comment (robinpiets) [inline packages/client/src/components/Outcome/NewCheckerModal.test.tsx:102]: irrelevant

File: packages/client/src/components/Outcome/OutcomeContent.test.tsx [hunk 5]
```diff
@@ -69,7 +69,5 @@ describe("OutcomeContent", () => {
     // Shouldn't be in document
     expect(screen.queryByTestId(NEED_PERMIT)).not.toBeInTheDocument();
     expect(screen.queryByTestId(NEED_CONTACT)).not.toBeInTheDocument();
-
-    // @TODO: Write test for PRINT_BUTTON
```
Comment (robinpiets) [inline packages/client/src/components/Outcome/OutcomeContent.test.tsx:73]: irrelevant

File: packages/client/src/components/Question/Question.tsx [hunk 6]
```diff
@@ -59,7 +59,6 @@ const Question: FunctionComponent<QuestionProps> = ({
 
   const isRadio = type === "boolean" || (type === "string" && options);
 
-  // @TODO: make generic functions, like: isRadioQuestion(), isCheckboxQuestion, etc
```
Comment (robinpiets) [inline packages/client/src/components/Question/Question.tsx:62]: useful when adding Checkbox and TextInput fields, not now

**Review by svenjens — APPROVED** [review]
🙌

**Pull request merged**